backend/app/literature/crud/mesh_detail_crud.py

```diff
@@ -43,12 +43,13 @@ def destroy(db: Session, mesh_detail_id: int) -> None:
 
 
 def patch(db: Session, mesh_detail_id: int, mesh_detail_update: MeshDetailSchemaUpdate) -> dict:
+    mesh_detail_data = jsonable_encoder(mesh_detail_update)
```

Comment:
I would also check with @ianlongden  to make sure that the patch functions work with the generic update functions that he made. I am not sure if we can use his functions in this context. Or maybe we should be extending what he has already done. I think his functions are only for adding things related to reference/resource.